editor/modes/visual-editor/style.scss

```diff
@@ -59,7 +59,6 @@
 	&.has-warning .editor-visual-editor__block-edit {
 		position: relative;
 		min-height: 250px;
-		pointer-events: none;
```

Comment:
I had to remove this to fix clicking on the buttons. Any reason for this @aduth?

Reply:
I think the original reason was to prevent people from interacting with the preview _behind_ the overlay.

Reply:
This regressed in #2618 with moving the warning into the invalid block.

https://github.com/WordPress/gutenberg/pull/2618/files#diff-5cb891cd3125ad3b221777433a61a524

The style should only apply to the preview, not the warning.

Reply:
This change can be omitted during rebase given #2811.